.eslintrc.cjs

```diff
@@ -117,5 +117,13 @@ module.exports = {
         'no-console': 'off',
       },
     },
+    {
+      files: ['packages/astro/src/core/errors/errors-data.ts'],
+      rules: {
+        // This file is used for docs generation, as such the code need to be in a certain format, we can somewhat ensure this with these rules
```

Comment:
Those two rules are unfortunately not perfect, for a real perfect dx, we'd need to take into account the JSDoc comments and stuff. Nonetheless, this prevents `message: (something) => {return something}`, which is what I mostly wanted.